ui/pages/onboarding-flow/onboarding-flow.js

```diff
@@ -161,6 +163,15 @@ export default function OnboardingFlow() {
             path={ONBOARDING_METAMETRICS}
             component={MetaMetricsComponent}
           />
+          <Route
+            path={ONBOARDING_EXPERIMENTAL_AREA}
+            render={(routeProps) => (
+              <ExperimentalArea
+                {...routeProps}
+                redirectTo={ONBOARDING_WELCOME_ROUTE}
+              />
+            )}
+          />
```

Comment:
In this file, this `Route` component should be fenced so that it is only included in `flask` builds. Any imports only used in these lines must also be fenced, or builds will fail due to lint errors. They are probably failing for this branch at the moment but a bug is causing the build job to still exit with zero. Confirm that everything is working by running `yarn start --build-type flask` locally.